README.md

```diff
@@ -1,60 +1,78 @@
-# ROCm™ v5.2.3
-This repository contains the manifest file for ROCm™ releases, changelogs and release information. The file default.xml contains information all the repositories and the associated commit use to build the current ROCm release. The default.xml file uses the repo Manifest format.
+# ROCm™ Repository Updates
+This repository contains the manifest file for ROCm™ releases, changelogs, and release information. The file default.xml contains information for all repositories and the associated commit used to build the current ROCm release.
 
-# Release Notes
-The ROCm v5.2.3 patch release is now available. The details are listed below. Highlights of this release include a bump in RCCL version compatibility and minor bug fixes in the HIP Runtime. Additionally, ROCm releases will return to use of the 
-[ROCm](https://github.com/RadeonOpenCompute/ROCm) repository for version controlled release notes henceforth. This 
-release of ROCm is validated with the AMDGPU release 22.20.1.
+The default.xml file uses the repo Manifest format.
 
-All users of the ROCm v5.2.1 release and below are encouraged to upgrade. Refer to https://docs.amd.com for all documentation associated with this release. 
+# ROCm v5.2.3 Release Notes
+The ROCm v5.2.3 patch release is now available. The details are listed below. Highlights of this release include enhancements in RCCL version compatibility and minor bug fixes in the HIP Runtime.
 
+Additionally, ROCm releases will return to use of the [ROCm](https://github.com/RadeonOpenCompute/ROCm) repository for version-controlled release notes henceforth.
 
-## All Components
-### Ubuntu 18.04 End of Life Announcement
-ROCm v5.2.3 is the last release to support Ubuntu 18.04. Future releases of ROCm will not provide prebuilt packages for Ubuntu 18.04. 
+**NOTE**: This release of ROCm is validated with the AMDGPU release v22.20.1.
 
-## HIP and Other Runtimes
+All users of the ROCm v5.2.1 release and below are encouraged to upgrade. Refer to https://docs.amd.com for documentation associated with this release.
 
-### HIP Runtime
 
-#### Fixes
- - Fixed a bug discovered in the HIP graph capture implementation in the ROCm v5.2.0 release. If the same kernel is called twice
- (with different argument values) in a graph capture, the implementation was only keeping the argument values for 
- the second kernel call.
- - Fixed a bug introduced in the hiprtc implementation in the ROCm v5.2.0 release. This bug caused the *hiprtcGetLoweredName* call failed
- for named expressions with a whitespace in it. 
+## Introducing Preview Support for Ubuntu 22.04.5 HWE
```

Comment:
"22.04.5 HWE"?
There are only
20.04.5 HWE
22.04.1
Unclear which one you refers to maybe both. Please correct the message.

Reply:
@ye-luo Thank you for the feedback; we appreciate your letting us know.

We have corrected the typo to reflect Ubuntu 20.04.5 HWE. 

ROCm Documentation Team
https://docs.amd.com